x/finality/keeper/tallying_bench_test.go

```diff
@@ -82,7 +83,7 @@ func benchmarkTallyBlocks(b *testing.B, numFPs int) {
 
 		b.StartTimer()
 
-		fKeeper.TallyBlocks(ctx)
+		fKeeper.TallyBlocks(ctx, math.MaxUint64)
```

Comment:
Will `startHeight+maxFinalizedBlocks` overflow if `maxFinalizedBlocks` is `math.MaxUint64`?

Reply:
Yep will fix the test